src/components/atoms/TCNavbarItem/TCNavbarItem.tsx

```diff
@@ -0,0 +1,52 @@
+import './TCNavbarItem.scss'
+import { Link } from 'react-router-dom'
+import { NavbarItemModel } from '../../../model/NavbarItemModel'
+import { Icons } from '../../../assets/icons/Icons'
+import { FunctionComponent } from 'react'
+
+interface TCNavbarItemProps {
+  item: NavbarItemModel
+  onClick: (itemIndex: number) => void
+  selectedItemIndex: number
+  index: number
+}
+
+export const TCNavbarItem: FunctionComponent<TCNavbarItemProps> = ({
+  item,
+  onClick,
+  selectedItemIndex,
+  index,
```

Comment:
Instead of passing index and selectedItemIndex can we pass isSelected as a boolean?